database/buildSchema/34B_review_assignable_functions.sql

```diff
@@ -0,0 +1,143 @@
+-- Function to return elements reviewable questions (per application)
```

Comment:
You have to be careful with numbering "B", as the shell script doesn't do a good job with ordering (I think it puts 34B before 34). When I save core_templates with this script and re-run it, it throws an error because of this ordering problem. (I dealt with this a lot way back when we were using these scripts all the time.)

I *think* it's okay to go 34A then 34B, but I'd probably play it safe and just reshuffle all the subsequent numbers so there's no letters in the numbering.

Reply:
Thanks! I have done this on the new base PR - Going to base this PR shortly!